src/python/grpcio/grpc/_cython/_cygrpc/aio/completion_queue.pyx.pxi

```diff
@@ -0,0 +1,106 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef gpr_timespec _GPR_INF_FUTURE = gpr_inf_future(GPR_CLOCK_REALTIME)
+
+
+def _handle_callback_wrapper(CallbackWrapper callback_wrapper, int success):
+    CallbackWrapper.functor_run(callback_wrapper.c_functor(), success)
+
+
+cdef class BaseCompletionQueue:
+
+    async def shutdown(self):
+        raise NotImplementedError()
+
+    cdef grpc_completion_queue* c_ptr(self):
+        raise NotImplementedError()
+
+
+cdef class PollerCompletionQueue(BaseCompletionQueue):
+
+    def __cinit__(self):
+        self._cq = grpc_completion_queue_create_for_next(NULL)
+        self._shutdown = False
+        self._shutdown_completed = asyncio.get_event_loop().create_future()
+        self._poller = None
+        self._poller_running = asyncio.get_event_loop().create_future()
+        self._poller = threading.Thread(target=self._poll_wrapper)
+        self._poller.daemon = True
+        self._poller.start()
+
+    cdef void _poll(self) except *:
+        cdef grpc_event event
+        cdef CallbackContext *context
+        cdef object waiter
+        grpc_call_soon_threadsafe(self._poller_running.set_result, None)
```

Comment:
I don't know you but for me every time that I read `grpc_call_*` makes me think that is an internal function provided by the core ...

It's not the first one, and there are others and are most of them in this module, would make sense name them with another prefix, like `aio_iomgr_`?  or `aio_loop_`?

Reply:
Changed to `aio_loop` prefix. It might change later with the support for many-thread many-loop.